src/main/java/competition/subsystems/hood/HoodSubsystem.java

```diff
@@ -0,0 +1,65 @@
+package competition.subsystems.hood;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.math.MathUtils;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+
+@Singleton
+public class HoodSubsystem extends BaseSubsystem{
+
+    final DoubleProperty extendPowerProp;
+    final DoubleProperty retractPowerProp;
+    final DoubleProperty maxAngleProp;
+    final DoubleProperty minAngleProp;
+    final DoubleProperty curAngleProp;
```

Comment:
go ahead and write it as `currentAngleProp` - readability is more important than small variables.